buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -96,7 +96,7 @@ object Versions {
 
   const val barista: String = "2.7.1" // available: "3.7.0"
 
-  const val xfetch2: String = "3.1.6"
```

Comment:
We need to remove `xfetch2okhttp`  string as well.

Reply:
done